django_icons/renderers/image.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+from django.conf import settings
+from django.forms.utils import flatatt
+from django.utils.html import format_html
+from django.utils.safestring import mark_safe
+
+from django_icons.css import merge_css_text
+from django_icons.renderers.base import BaseRenderer
+
+
+class ImageRenderer(BaseRenderer):
+    """
+    Render an icon from a local file
+
+    <img src="/static/icons/icon.png'" class="icon icon-{name}" %}">
+    """
+
+    def get_class(self):
+        """
+        First class, usually defines the icon
+        """
+        return 'icon icon-{name}'.format(name=self.name)
+
+    def get_icon_prefix(self):
+        """
+        Filename prefix. For example, Icons8 icons are all prefixed with `icons8-`, so that you can call your icons
+        without repeating the prefix in the name.
+        """
+        return ''
+
+    def get_prefix(self):
+        """
+        Path prefix
+        """
+        return 'icons'
```

Comment:
Not quite sure about this. If I was defining my icons and they were in images that were NOT in `STATIC/icons` I'd have to write my own renderer and overwrite this. If this was empty, I could define all my icons by adding my path from static, which seems more logical. I think adding an example to the demo might help to figure this one out.

Reply:
Based on my use case, I indeed made the assumption that the icons would located in `STATIC/icons`. I think we have different logics here, as I prefer to write my own Renderer to override what I need to customize instead of adding the path in the template tag call.